cpp/modmesh/view/wrap_view.cpp

```diff
@@ -32,16 +32,19 @@
 
 #include <modmesh/view/view.hpp>
 
+#include <QPointer>
 #include <QClipboard>
 
+PYBIND11_DECLARE_HOLDER_TYPE(T, QPointer<T>);
+
 namespace modmesh
 {
 
 namespace python
 {
 
 class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapR3DWidget
-    : public WrapBase<WrapR3DWidget, R3DWidget>
+    : public WrapBase<WrapR3DWidget, R3DWidget, QPointer<R3DWidget>>
```

Comment:
Without the `QPointer` holder, sometimes viewer crashes because of dangling pointer to Qt.